composer.json

```diff
@@ -4,33 +4,51 @@
     "description": "Easycredit module",
     "license": "MIT",
     "require": {
-        "php": ">=7.2",
+        "php": ">=7.4",
+        "spryker-shop/checkout-page": "^3.21.0",
+        "spryker-shop/shop-application": "^1.13.0",
+        "spryker/calculation": "^4.12.0",
         "spryker/checkout-extension": "^1.0.0",
         "spryker/guzzle": "^2.2.0",
         "spryker/kernel": "^3.31.0",
-        "spryker/sales": "^8.17.0 || ^10.0.0 || ^11.0.0",
```

Comment:
It looks like a BC break. Was it approved by TechLead?

Reply:
No. Changed it back and added `^2.2.0 ||` version for `spryker-shop/checkout-page`